### PR TITLE
Update contents and hyperlinks

### PR DESCRIPTION
**Pull request by giddygarcia**
- Update text contents to latest mockup  
- Apply hyperlinks

### CODE REVIEW

**Comment by giddygarcia** [comment]
@Hjkun77

File: src/components/Ampatuan/Ampatuan.jsx [hunk 1]
```diff
@@ -13,45 +13,44 @@ const Ampatuan = () => (
...
... <a href="https://cnnphilippines.com/news/2019/12/20/Maguindanao-massacre-women-rape-sperm-.html?fbclid=IwAR1Efyt6JK9TIRFJM5qbO40hcEk4bmwrug0wIyfExAc82TxSfSFgT2Wq-aU">raped</a> before being shot. Nearly all female victims were <a href="https://gulfnews.com/world/asia/philippines/official-confirms-mayor-is-the-suspect-in-political-massacre-1.533131">shot</a> in their genitalia. The local police were found to have been involved in the killings, being ordered by senior members of the <a href="https://gulfnews.com/world/asia/philippines/official-confirms-mayor-is-the-suspect-in-political-massacre-1.533131">Ampatuan clan</a> to carry out and cover up the massacre. The perpetrators were unable to finish clean up operations as the area was spotted by a <a href="https://news.abs-cbn.com/news/12/19/19/look-back-duterte-helped-search-for-maguindanao-massacre-site">helicopter</a>. All but one of the bodies of the victims were found. The Ampatuan massacre is often remembered as the <a href="http://nwasianweekly.com/2009/12/world%E2%80%99s-deadliest-assault-on-journalists-occur-in-philippines/">deadliest attack on the press in history</a>. The massacre was also an instance of election violence.
```
Comment (Hjkun77) [inline src/components/Ampatuan/Ampatuan.jsx]: `target="_blank" rel="noopener noreferrer"`

File: src/components/Ampatuan/Ampatuan.jsx [hunk 2]
```diff
@@ -13,45 +13,44 @@ const Ampatuan = () => (
...
... years after the Ampatuan massacre, Judge Jocelyn Solis-Reyes passed a <a href="https://edition.cnn.com/2019/12/18/asia/philippines-maguindanao-massacre-verdict-intl-hnk/index.html">verdict</a> for the suspects in police custody on December 18, 2019. The court found 43 of the <a href="https://news.abs-cbn.com/news/12/19/19/list-qc-courts-verdict-on-maguindanao-massacre-suspects">suspects</a> guilty on <a href="https://www.rappler.com/nation/247517-58th-ampatuan-massacre-victim-forgotten-verdict">57 counts</a> of murder as the 58th victim's body was never found. The accused faced the charge of <u>57 counts of murder</u>, one count for each victim. In January 2020, <a href="https://news.abs-cbn.com/news/01/31/20/2-more-maguindanao-massacre-suspects-plead-not-guilty-to-multiple-murder-raps">two more suspects</a> were identified and arrested by the police. Listed below are the journalists killed in the massacre as listed by the <a href="https://cpj.org/data/killed/asia/philippines/?status=Killed&motiveConfirmed%5B%5D=Confirmed&motiveUnconfirmed%5B%5D=Unconfirmed&type%5B%5D=Journalist&cc_fips%5B%5D=RP&start_year=2009&end_year=2009&group_by=location">Committee to Protect Journalists</a>.
```
Comment (Hjkun77) [inline src/components/Ampatuan/Ampatuan.jsx]: `target="_blank" rel="noopener noreferrer"`

File: src/components/Ampatuan/Ampatuan.jsx [hunk 3]
```diff
@@ -13,45 +13,44 @@ const Ampatuan = () => (
...
-        <div>Ronnie Perante</div>
-        <div>Joel Parcon</div>
-        <div>Fernando Razon</div>
-        <div>Alejandro Reblando</div>
-        <div>Napoleon Salaysay</div>
-        <div>Ian Subang</div>
-        <div>Andres Teodoro</div>
-        <div></div>
-        <div></div>
+      <div><a href="https://cpj.org/data/people/henry-araneta/index.php">Henry Araneta</a></div>
```
Comment (Hjkun77) [inline src/components/Ampatuan/Ampatuan.jsx]: `target="_blank" rel="noopener noreferrer"`

File: src/components/JournalistBox/JournalistBox.jsx [hunk 4]
```diff
@@ -4,7 +4,10 @@ import { Item, Event, Name, Desc } from "./styles"
 const JournalistBox = ({ event, name, date, place, description }) => (
   <Item>
     <Event>{event}</Event>
-    <Name>{name}</Name>
+    <Name>
+      <a href={link} target="_blank" target="_blank"
```
Comment (Hjkun77) [inline src/components/JournalistBox/JournalistBox.jsx]: remove `target="_blank"`

**Review by Hjkun77 — CHANGES_REQUESTED** [review]

**Pull request merged**